set.mm

```diff
@@ -85685,6 +85710,101 @@ the first case of his notation (simple exponentiation) and subscript it
       IJK $.
   $}
 
+  ${
+    $d B b f g $.  $d S b f g $.  $d V b g $.
+    $( The class of all functions from a (proper) singleton into ` B ` is the
+       union of all the singletons of (proper) ordered pairs over the elements
+       of ` B ` as second component.  (Contributed by AV, 13-Sep-2024.) $)
+    fsetsniunop $p |- ( S e. V -> { f | f : { S } --> B }
+                                  = U_ b e. B { { <. S , b >. } } ) $=
+      ( vg wcel csn cv wf cab cop ciun wrex wceq cfv wa fsn2g simpl simpr opeq2
+      wb sneqd eqeq2d adantl rspcedvd syl6bi fsnd adantr feq1d mpbird rexlimdva
+      ex impbid velsn bicomi rexbii bitrdi vex feq1 elab eliun 3bitr4g eqrdv )
+      BDGZFBHZACIZJZCKZEABEIZLZHZHZMZVEVFAFIZJZVOVMGZEANZVOVIGVOVNGVEVPVOVLOZEA
+      NZVRVEVPVTVEVPBVOPZAGZVOBWALZHZOZQZVTBAVODRWFVSWEEWAAWBWESVJWAOZVSWEUBWFW
+      GVLWDVOWGVKWCVJWABUAUCUDUEWBWETUFUGVEVSVPEAVEVJAGZQZVSVPWIVSQZVPVFAVLJZWI
+      WKVSWIBVJDAVEWHSVEWHTUHUIWJVFAVOVLWIVSTUJUKUMULUNVSVQEAVQVSFVLUOUPUQURVHV
+      PCVOFUSVFAVGVOUTVAEVOAVMVBVCVD $.
+
+    $d B b y $.  $d S y $.
+    $( The class of all functions from a (proper) singleton into ` B ` is the
+       set of all the singletons of (proper) ordered pairs over the elements of
+       ` B ` as second component.  (Contributed by AV, 13-Sep-2024.) $)
+    fsetabsnop $p |- ( S e. V -> { f | f : { S } --> B }
+                                 = { y | E. b e. B y = { <. S , b >. } } ) $=
+      ( wcel csn cv wf cab cop ciun wceq wrex fsetsniunop iunsn eqtrdi ) CEGCHB
+      DIJDKFBCFILHZHMAISNFBOAKBCDEFPFABSQR $.
+  $}
+
+  ${
+    $d A f g $.  $d B f g $.  $d F g $.  $d X g $.
+    fsetfocdm.f $e |- F = { f | f : A --> B } $.
+    fsetfocdm.s $e |- S = ( g e. F |-> ( g ` X ) ) $.
+    $( The mapping of an element of a class of functions to a fixed function
+       value is a function.  (Contributed by AV, 15-Sep-2024.) $)
```

Comment:
The phrase "evaluation at X" should appear in this comment. For instance:

> The class of functions with a given domain and a given codomain is mapped, through evaluation at a point of the domain, into the codomain.

Reply:
Done